src/tls/Makefile.Kremlin

```diff
@@ -4,21 +4,24 @@ build: build-internal-test build-library build-external-test
 
 test: test-internal-test # test-library test-external-test
 
-clean: clean-internal-test clean-library clean-external-test
+ifneq (,$(MITLS_USE_OPENSSL))
+CONCRETE_FLAVOR	= -OpenSSL
+endif
 
 # To include Makefile.common, you need to specify
-# the following three variables
+# the following five variables
 FLAVOR		= Kremlin
 EXTENSION	= krml
 # Don't extract modules from mitls that are implemented in C
 EXTRACT		= '* -DHDB -LowCProvider -FFICallbacks -BufferBytes'
-
-ifneq (,$(MITLS_USE_OPENSSL))
-CONCRETE_FLAVOR	= -OpenSSL
-endif
+SPECINC     = $(MITLS_HOME)/src/tls/concrete-flags $(MITLS_HOME)/src/tls/concrete-flags/$(FLAVOR)$(CONCRETE_FLAVOR)
+VFLAGS     ?= --admit_smt_queries true
 
 include Makefile.common
 
+clean: clean-internal-test clean-library clean-external-test
```

Comment:
this defers cleaning to the sub-Makefiles in the extract/Kremlin-* directories, which is good; what do you think the behavior should be there? remove the .o files only, or also the .c files?

speaking of which, the `clean` command in these sub-directories is not correct, since it does not catch .o files in sub-directories; can you fix it while we're at it?